python-backend/tests/status_code_gen.py

```diff
@@ -95,3 +96,11 @@ def SampleDangerousOccurrenceSubparagraphs(num):
         db.session.query(ComplianceArticle).filter(
             ComplianceArticle.section == '1', ComplianceArticle.sub_section == '7',
             ComplianceArticle.paragraph == '3', ComplianceArticle.sub_paragraph != None).all(), num)
+
+def RandomVarianceApplicationStatusCode():
+    return random.choice([
+        x.variance_application_status_code
+        for x in filter(
+            lambda x: x.variance_application_status_code not in ['APP', 'DEN'],
+            VarianceApplicationStatusCode.active())
```

Comment:
Is this setting the status to active when the variance is not app or denied?

Reply:
This is just using the `.active()` method, which is a custom method on the `VarianceApplicationStatusCode` model, to run a query for all active variance application status code options. Then, it filters 'APP' and 'DEC' out of the list, returning all the other codes.

The reason for filtering these out is that approved and denied variances have requirements for other column values. This is part of the tests factory code, so you can see how it's used and how APP & DEC are handled in `factories.py`